src/main/java/de/hysky/skyblocker/utils/Location.java

```diff
@@ -82,6 +82,10 @@ public enum Location {
      * mode: kuudra
      */
     KUUDRAS_HOLLOW("kuudra"),
+    /**
+     * The freezing cold Glacite Mineshafts! *brr... so cold... :(*
+     */
```

Comment:
lol

Reply:
I wanted to spice up the description :')